cli/tests/integrations/test_marathon_pod.py

```diff
@@ -85,14 +88,19 @@ def test_pod_update_from_stdin_force_true():
 
 def _pod_add_from_file(file_path):
     cmd = _POD_ADD_CMD + [file_path]
-    return exec_command(cmd)
+    outputs = exec_command(cmd)
+
+    watch_all_deployments()
+    return outputs
```

Comment:
just this one. since we're returning all outputs of `exec_command` it alludes that we're doing the checking of the values (and thus `watch_all_deployments`) elsewhere